packages/web/tests/components/DiscordCheck/DiscordButtonCheck.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import fetchMock from 'fetch-mock-jest';
+import { DiscordButton } from '../../../src/components/DiscordCheck';
+import { render, screen, waitFor } from '../../testUtils/testTools';
+
+interface User {
+  discordId?: string;
+}
+
+const sampleUser: User = {
```

Comment:
By creating a new `User` interface, we're losing any parity with the *actual* type and we lose any value we'd normally gain from using a type, so we might as well just use `any`. Instead, let's use `Partial` so any keys we provide are validated against the actual type; if they differ or the key doesn't exist, TypeScript will get angry and we'll need to fix our tests (which is a great thing).
```suggestion
const sampleUser: Partial<User> = {
```